src/dbnode/storage/series/series.go

```diff
@@ -690,19 +574,19 @@ func (s *dbSeries) Close() {
 		// by the WiredList and should not be closed here. They will eventually
 		// be evicted and closed by  the WiredList when it needs to make room
 		// for new blocks.
-		for _, block := range s.blocks.AllBlocks() {
+		for _, block := range s.cachedBlocks.AllBlocks() {
 			if !block.WasRetrievedFromDisk() {
 				block.Close()
```

Comment:
Should we just remove this? There should never be anything in here that was not retrieved from disk so there should be nothing to close. Can maybe keep it in and instead of closing emit an invariant violated